### ISSUE TITLE
no template for modelDefine plugin

### ISSUE THREAD

**Issue by WenheLI** [alibaba/pipcook#362]
When running the following commands to generate a `modelDefine` plugin template:
```bash
pipcook plugin-dev --type modelDefine --name  tensorflow-rnn-model-define
```
the system will throw the following error:
```bash
Error: ENOENT: no such file or directory, copyfile '/usr/local/lib/node_modules/@pipcook/pipcook-cli/dist/assets/pluginPackage/src/modelDefine.ts'
```
suggesting the template for `modelDefine` does not exist.

**Comment by yorkie** [comment]
Thanks for the catch, BTW we have another plan to use the `npm-init` project instead of self-maintained templates at https://github.com/alibaba/pipcook/issues/154.

**Comment by WenheLI** [comment]
closed by #363
